rrmngmnt/operatingsystem.py

```diff
@@ -114,6 +116,58 @@ def distribution(self):
             self._dist = self.get_distribution()
         return self._dist
 
+    def get_kernel_info(self):
+        """
+        Get kernel info (release, version and type)
+
+        Returns:
+            namedtuple Kernel: Results tuple(release, version and type)
+
+        Examples:
+             kernel(
+                    release='4.18.0-135.el8.x86_64',
+                    version='#1 SMP Fri Aug 16 19:31:40 UTC 2019',
+                    type'='x86_64'
+                    )
+        """
+        values = ["release", "version", "type"]
+        cmd = ["uname", "-r", ";", "uname", "-v", ";", "uname", "-m"]
+        out = self._exec_command(
+            cmd=cmd, err_msg="Failed to obta kernel info"
+        )
+        Kernel = namedtuple('Kernel', values)
```

Comment:
kernel, should be all lowercase

Reply:
made it from capital just to be aligned with already existing code (see Distribution in get_distribution method)

Reply:
This is wrong, but I guess @lukas-bednar should decide.

Reply:
I believe this is alright. `namedtuple` returns a class, and then you create instance of that class. And classes in python should be Uppercased.